### DISCUSSION TITLE
Add support for Backup/recovery codes as MFA

### DISCUSSION THREAD

**Discussion by slengo8** [supabase/supabase#16068] [Feature Requests]
Hey!
Currently the only type of MFA allowed is TOTP factors

If the user loses their device they have the MFA setup on, they only way to recover form this is for them to contact the business owner which can be long. 
It would be beneficial if there is some other step that the user can handle this by themselves before they reach to the business owner

Looking forward to hearing form you
Have a good one 😃

**Comment by rienheuver** [comment]
Much agreed with this request! Self-implemented solutions for this are not favoured because it can easily corrupt the security. This would be a great addition

**Comment by pedrosimao** [comment]
I strongly support this one. I am implementing MFA now and I am negatively surprised no back-up codes / recovery methods are offered.
